frontend/src/components/Review/InfoFeatures.tsx

```diff
@@ -19,16 +18,14 @@ export default function InfoFeatures({
   address,
 }: Props): ReactElement {
   return (
-    <Grid item xs={12} sm={4}>
-      <Card variant="outlined">
-        <CardContent>
-          <Info phone={phone} address={address} />
-          <div className={styles.divider}></div>
-          <PropertyInfo title="Property Info" info={propertyInfo} />
-          <div className={styles.divider}></div>
-          <PropertyInfo title="Property Features" info={propertyFeatures} />
-        </CardContent>
-      </Card>
-    </Grid>
+    <Card variant="outlined">
+      <CardContent>
+        <Info phone={phone} address={address} />
+        <div className={styles.divider}></div>
```

Comment:
this is an element with no contents, so you can just do `<div ... />`

Reply:
ah this is not code that I wrote, I think MUI already has a <Divider /> so I'll use that instead